package.json

```diff
@@ -36,14 +36,14 @@
     "np": "^2.16.0",
     "npm-install-webpack-plugin": "^4.0.0",
     "prettier": "^1.3.1",
-    "prop-types": "15.5.10",
+    "prop-types": "15.6.0",
     "qs": "6.5.0",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
     "react-hot-loader": "^3.0.0-beta.7",
     "react-redux": "latest",
```

Comment:
Can you replace "latest" with the latest stable version of redux and check it works? `^5.0.6`?